### PR TITLE
Add '360 Panoroma' as new image type (TAM-135)

### PR DESCRIPTION
**Pull request by urshala**
Customer wants to render 360 degrees images in varaamo. Therefore,
add a new image type (360 Panoroma) to differentiate 360 degrees
images from normal 2d images.

### CODE REVIEW

File: resources/models/resource.py [hunk 1]
```diff
@@ -749,6 +749,7 @@ class ResourceImage(ModifiableModel):
         ('ground_plan', _('Ground plan')),
         ('map', _('Map')),
         ('other', _('Other')),
+        ('panoroma', _('360 panoroma')),
```
Comment (wiltso) [inline resources/models/resource.py]: 'panorama', _('360 panorama')

File: locale/fi/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -1140,6 +1140,9 @@ msgstr "Kartta"
 msgid "Other"
 msgstr "Toinen"
 
+msgid "360 panoroma"
+msgstr ""
```
Comment (wiltso) [inline locale/fi/LC_MESSAGES/django.po]: 360 panoraama

File: locale/sv/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -1136,6 +1136,9 @@ msgstr "Karta"
 msgid "Other"
 msgstr "Annan"
 
+msgid "360 panoroma"
+msgstr ""
```
Comment (wiltso) [inline locale/sv/LC_MESSAGES/django.po]: 360 panorama

File: resources/migrations/0098_add_panoroma_image_type.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.11 on 2022-06-08 08:31
```
Comment (wiltso) [inline resources/migrations/0098_add_panoroma_image_type.py]: Recreate this when you have updated the string.

File: resources/models/resource.py (hunk 1 above)
Comment (urshala) [reply resources/models/resource.py]: Changed it.

File: locale/fi/LC_MESSAGES/django.po (hunk 2 above)
Comment (urshala) [reply locale/fi/LC_MESSAGES/django.po]: Changed it.

File: locale/sv/LC_MESSAGES/django.po (hunk 3 above)
Comment (urshala) [reply locale/sv/LC_MESSAGES/django.po]: Changed it.

File: resources/migrations/0098_add_panoroma_image_type.py (hunk 4 above)
Comment (urshala) [reply resources/migrations/0098_add_panoroma_image_type.py]: Updated the migrations file.

**Review by wiltso — APPROVED** [review]
Looks good! Make sure the tests have passed before merging

**Pull request merged**